railties/test/application/test_runner_test.rb

```diff
@@ -430,6 +430,26 @@ class PostTest < ActiveSupport::TestCase
       end
     end
 
+    def test_unknown_option_is_invalid
+      create_test_file
+
+      assert_match 'invalid option: --unknown', run_test_command('--unknown')
+    end
+
+    def test_extra_plugin_options_goes_through_unscathed
+      skip 'cannot get this to load the plugin'
```

Comment:
This works locally, but I can't figure out how to simulate a plugin in the pipeline properly. Ideas welcome 😁